docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -35,6 +38,33 @@ As of CentOS 8, `ntp` is no longer available and has been replaced by `chrony`.
 $ sudo yum install -y chrony
 ```
 
+### PHC (PTP Hardware Clock) {{<tags/feature/tp>}}
+
+Hardware clocks are more accurate.
+
+See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configure-ec2-ntp.html#connect-to-the-ptp-hardware-clock
+to check whether your AWS instance supports PHC.
+
+Configure PTP using
+
+```sh
+sudo bash ./bin/configure_ptp.sh
+```
+
+### Clockbound Agent {{<tags/feature/tp>}}
+
+Configuring a clockbound agent improves clock accuracy significantly.
+PHC is desirable but not mandatory.
+Chrony is a pre-requisite.
```

Comment:
While PHC is beneficial, it is not required. However, Chrony is a necessary prerequisite.

Reply:
Done